app/components/meetings/upload-statement-dialog.vue

```diff
@@ -255,6 +256,18 @@ export default {
 
             this.meetings = this.meetings.filter(o=>o.uploadStatement);
 
+            if(this.filterByMeetingAgenda){
+                console.log(this.filterByMeetingAgenda)
+                const filterMeetings = Object.keys(this.filterByMeetingAgenda);
+                this.meetings = this.meetings
+                                .filter(o=>{ //filter by meetings
+                                    return filterMeetings.includes(o.EVT_SHT_NM)
```

Comment:
If you want to mean meeting code you have to use `normalizedSymbol `
```suggestion
                                    return filterMeetings.includes(o. normalizedSymbol)
```